ui/src/components/CountrySelector.js

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2020 Google LLC
+ 
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+ 
+    https://www.apache.org/licenses/LICENSE-2.0
+ 
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+ 
+import {LitElement, html, css} from 'lit-element';
+
+import {CountryService} from '../utils/CountryService';
+
+import {Select} from '@material/mwc-select';
+import {ListItem} from '@material/mwc-list/mwc-list-item';
+ 
+/**
+ * Component responsible for providing users a means to select
+ * the country they are located in. 
+ */
+export class CountrySelector extends LitElement {
+
+    static get properties() {
+        return {
+            countries: {type: Array}
+        };
+    }
+
+    static get styles() {
+        return css`
+            div {
+                align-items: center; 
+                display: flex; 
+                flex-direction: column; 
+                flex-wrap: wrap;
+                justify-content: center; 
+                text-align: center;  
+            }
+            h3 {
+                font-family: 'Roboto';
+                font-size: 30px; 
+                font-weight: normal;
+            }
+            p {
+                font-family: 'Roboto';
+                font-size: 20px; 
+                font-weight: bolder; 
+            }
+        `;
+    }
```

Comment:
fyi this'll get moved to its own file at some point once I explain to y'all how styling's set up

Reply:
I appreciate the heads up--I might ask for your help with the merge conflict when this pr is ready